src/app/lib/config/services/zhongwen.ts

```diff
@@ -239,7 +239,7 @@ export const service: ZhongwenConfig = {
           },
           accept: '可以',
           reject: '了解更新内容',
-          rejectUrl: 'https://www.bbc.com/usingthebbc/your-data-matters',
+          rejectUrl: 'https://www.bbc.com/usingthebbc/privacy-policy/',
```

Comment:
As for serbian - do we need another entry for simp (or trad? can't remember which one)